Resources/Prototypes/Entities/Clothing/Uniforms/scrubs.yml

```diff
@@ -0,0 +1,32 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: UniformScrubsPurple
+  name: purple scrubs
```

Comment:
Should these have colour on them? Just because stuff like the bio suit for example doesn't it.